tests/test_shell.rs

```diff
@@ -60,17 +60,20 @@ test!(colored_shell {
                                             color::RED).unwrap()));
 });
 
-test!(quiet_shell {
-    let config = ShellConfig { color: true, verbosity: Quiet, tty: true };
```

Comment:
Same as the previous comment. `Shell` no longer has a concept of verbosity so this test didn't have anything left to test.